### PR TITLE
Milestone 17: complete project

### PR DESCRIPTION
**Pull request by Lizdev-05**
In this milestone, we achieved the following...

- Completed the config & basic setup
- Fetched data and updated Redux store
- dispatched an action to store the selected data in the Redux store
- Rendered user interface
-  Wrote actions and reducers for booking rockets and joining missions
- Wrote actions and reducers for canceling rockets and leaving missions
- Rendered UI through conditional components rendering
- Listed the rockets reserved and missions joined by the user (as per design) in the UI
- Set up testing libraries(React Testing Library and Jest).

### CODE REVIEW

File: src/components/pages/RockectDisplay.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+/* .main button {
+  background-color: rgb(17, 116, 255);
+  padding: 0.5rem 1rem;
+  outline: none;
+  border: none;
+  color: #fff;
+  border-radius: 0.3rem;
+} */
```
Comment (prolajumokeoni) [inline src/components/pages/RockectDisplay.css:29]: - [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉

File: src/components/pages/RockectDisplay.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable camelcase */
```
Comment (prolajumokeoni) [inline src/components/pages/RockectDisplay.js:2]: - [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿

File: src/redux/rockets/RockectReducer.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-unused-vars */
```
Comment (prolajumokeoni) [inline src/redux/rockets/RockectReducer.js:1]: - [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @hammaazarok  and @Lizdev-05 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/pages/RockectDisplay.js (hunk 2 above)
Comment (Lizdev-05) [reply src/components/pages/RockectDisplay.js:2]: Good day @prolajumokeoni, the disabled linters here were for prop-types, it is not accepting array whereas, the prop is an array so I need to use prop-types of array and for it to work I need to disable the eslint that is not allowing it. I won't mind getting views on how to go about it from you. If there is another way to do that, I will like to jump on a zoom call with you for the explanation. Thank you so much.

File: src/components/pages/RockectDisplay.css (hunk 1 above)
Comment (Lizdev-05) [reply src/components/pages/RockectDisplay.css:29]: Noted

File: src/redux/rockets/RockectReducer.js (hunk 3 above)
Comment (Lizdev-05) [reply src/redux/rockets/RockectReducer.js]: Solved

File: src/components/pages/RockectDisplay.css (hunk 1 above)
Comment (Lizdev-05) [reply src/components/pages/RockectDisplay.css]: Solved

**Review by mwanawabangona — APPROVED** [review]
Hi, @Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

- [ ]  Kindly `note` that the node modules always have to added to the `gitignore files` please see this [article](https://sebhastian.com/git-ignore-node_modules/#:~:text=The%20node_modules%2F%20folder%20is%20the,t%20write%20to%20the%20repository.) for further understanding

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**